### PR TITLE
Update syntax for the saveTableToContext step

### PR DESCRIPTION
**Pull request by vkepin**

### CODE REVIEW

File: docs/modules/commons/pages/variables.adoc [hunk 1]
```diff
@@ -219,3 +219,30 @@ When the condition `#{eval(`${optionalPathToCheck:/defaultPath}` != `/forbiddenP
 |step                                                                |
 |When I go to the relative URL '${optionalPathToCheck:/defaultPath}' |
 ----
+
+=== Complex types variables
+
+Vividus provides the ability to work with not only primitive variables like numbers and strings but also supports complex types such as lists and maps.
```
Comment (valfirst) [inline docs/modules/commons/pages/variables.adoc:225]: our users are not technical, they might not know data structures, thus we should avoid describing tables via lists and maps

Comment (vkepin) [reply docs/modules/commons/pages/variables.adoc:225]: our users can use conditions and loops, evaluate expressions and JS scripts, but will they be frustrated by lists and maps?

Comment (vkepin) [reply docs/modules/commons/pages/variables.adoc:225]: updated

File: docs/modules/commons/pages/variables.adoc [hunk 2]
```diff
@@ -219,3 +219,29 @@ When the condition `#{eval(`${optionalPathToCheck:/defaultPath}` != `/forbiddenP
...
+
+==== Accessing values
+
+* Values from list variables can be accessed by index. For example, `+${listVar[0]}+` returns the first value.
+* Values from map variables can be accessed by key. For instance, `+${listVar.key1}+` returns the value associated with 'key1'.
```
Comment (valfirst) [inline docs/modules/commons/pages/variables.adoc:230]:
```suggestion
* Values from map variables can be accessed by key. For instance, `+${mapVar.key}+` returns the value associated with 'key'.
```

File: docs/modules/commons/pages/variables.adoc [hunk 3]
```diff
@@ -219,3 +219,29 @@ When the condition `#{eval(`${optionalPathToCheck:/defaultPath}` != `/forbiddenP
 |step                                                                |
 |When I go to the relative URL '${optionalPathToCheck:/defaultPath}' |
 ----
+
+=== Complex types variables
+
+Vividus provides the ability to work with not only primitive variables like numbers and strings but also supports complex types such as lists and maps.
```
Comment (valfirst) [inline docs/modules/commons/pages/variables.adoc:225]:
```suggestion
VIVIDUS provides the ability to work with not only primitive variables like numbers and strings but also supports complex types such as lists and maps.
```

File: docs/modules/commons/pages/variables.adoc [hunk 4]
```diff
@@ -219,3 +219,29 @@ When the condition `#{eval(`${optionalPathToCheck:/defaultPath}` != `/forbiddenP
...
+
+==== Accessing values
+
+* Values from list variables can be accessed by index. For example, `+${listVar[0]}+` returns the first value.
+* Values from map variables can be accessed by key. For instance, `+${listVar.key1}+` returns the value associated with 'key1'.
+
+==== Working with tables
+
+Vividus provides the ability to store tables in variables.
```
Comment (valfirst) [inline docs/modules/commons/pages/variables.adoc:234]:
```suggestion
VIVIDUS provides the ability to store tables in variables.
```

File: docs/modules/commons/pages/variables.adoc [hunk 5]
```diff
@@ -219,3 +219,29 @@ When the condition `#{eval(`${optionalPathToCheck:/defaultPath}` != `/forbiddenP
...
+
+.Assuming we have the table below stored in the variable 'tableVar':
+[options="header",cols="2,2"]
+|===
+| A  | B
+| A1 | B1
+| A2 | B2
+|===
+
+To access specific cell values, specify the row and header key. Rows indexing started from 0.
```
Comment (valfirst) [inline docs/modules/commons/pages/variables.adoc:245]:
```suggestion
To access specific cell values, specify the row and header key. Rows indexing starts from 0.
```

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 6]
```diff
@@ -930,3 +930,32 @@ Then element located '$locator' disappears in '$timeout'
...
 Then element located 'By.xpath(//div[@id='delayed'])' disappears in 'PT3S'
 ----
+
+=== Set Variable Steps
+
+==== Save HTML table to the variable
+
+This step allows you to save an HTML table from a specified context into a variable.
+
+Values are stored in the variable as a list of maps. For more information, see xref:commons:variables.adoc#_working_with_tables[working with table variables]
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:940]:
```suggestion
For more information, see how xref:commons:variables.adoc#_working_with_tables[to work with table variables]
```

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 7]
```diff
@@ -930,3 +930,32 @@ Then element located '$locator' disappears in '$timeout'
...
+[source,gherkin]
+----
+When I save table to $scopes variable `$variableName`
+----
+
+* `$scopes` - xref:commons:variables.adoc#_scopes[The comma-separated set of the variables scopes].
+* `$variableName` - The variable name to refer to the variable value.
+
+[NOTE]
+Before using this step, ensure the test context is set to the required table. See the example below.
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:954]:
```suggestion
Before using this step, ensure the context is set to the required table. See the example below.
```

File: vividus-tests/src/main/resources/story/integration/SetVariableSteps.story [hunk 8]
```diff
@@ -1,58 +1,20 @@
...
+|A2|B2|C2|
+|A3|B3|C3|
+
+Scenario: Verify deprecated step: "When I save table to $scopes variable '$variableName'"
+Given I am on page with URL `${vividus-test-site-url}/table.html`
+When I change context to element located `xpath(//table)`
+When I save table to SCENARIO variable 'table'
+Then `${table[1]}` is equal to table ignoring extra columns:
+|A|C|
+|A2|C2|
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/SetVariableSteps.story:20]:
```suggestion
|A |C |
|A2|C2|
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**